src/mdformat/_api.py

```diff
@@ -68,7 +68,11 @@ def file(
     )
```

Comment:
Can't comment on the exact line, but if you look above, on line 61 we have
```python
original_md = f.read_text(encoding="utf-8")
```
We need to change this to
```python
original_md = f.read_bytes().decode()
```
in order to preserve line ending sequences.

Reply:
I will check this.

I also realsied I forget tests for the API side in generall.. 🤦🏼‍♂️